docs/core/entity/select.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Select Entity
+sidebar_label: Select
+---
+
+A `select` is an entity that allows the user to select an option from a list of limited options provided by the integration. Derive entity platforms from [`homeassistant.components.select.SelectEntity`](https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/select/__init__.py)
```

Comment:
Should we use the dev branch instead of master branch for the link?

Reply:
Hmmm it is a bit of a mixed back so it seems... but I agree, it should be using `dev` (and the new repo link format).